doc/getting-started/building-from-source.rst

```diff
@@ -238,53 +237,20 @@ corresponding VM configuration files in the corresponding
                 prompt "ACRN Scenario"
                 default SDC
 
-See the :ref:`hardware` document for information about the platform
-needs for each scenario.
-
-.. _getting-started-hypervisor-configuration:
-
-Modify the hypervisor configuration
-***********************************
-
-The ACRN hypervisor leverages Kconfig to manage configurations, powered by
-Kconfiglib. A default configuration is generated based on the board you have
-selected via the ``BOARD=`` command line parameter. You can make further
-changes to that default configuration to adjust to your specific
-requirements.
-
-To generate hypervisor configurations, you need to build the hypervisor
-individually. The following steps generate a default but complete configuration,
-based on the platform selected, assuming that you are under the top-level
-directory of acrn-hypervisor. The configuration file, named ``.config``, can be
-found under the target folder of your build.
-
-.. code-block:: none
-
-   $ make defconfig BOARD=nuc6cayh
-
-The BOARD specified is used to select a defconfig under
-``arch/x86/configs/``. The other command-line based options (e.g. ``RELEASE``)
-take no effects when generating a defconfig.
-
-To modify the hypervisor configurations, you can either edit ``.config``
-manually, or invoke a TUI-based menuconfig, powered by kconfiglib, by executing
-``make menuconfig``. As an example, the following commands, assuming that you
-are under the top-level directory of acrn-hypervisor, generate a default
-configuration file for UEFI, allow you to modify some configurations and build
-the hypervisor using the updated ``.config``.
+You can also change it directly from the command line.
 
 .. code-block:: none
 
-   $ make menuconfig              # Modify the configurations per your needs
+   $ cd  acrn-hypervisor/hypervisor
+   $ sed -i 's|default SDC|default INDUSTRY|' arch/x86/Kconfig
```

Comment:
Is this really the "approved" way to select a scenario?  Why aren't we using ``make menuconfig``?

Reply:
@ClaudZhang1995 , please don't delete make menuconfig, we can provide 2 ways to selectt one scenario. Thanks

Reply:
Specifying the scenario in menuconfig IS supported and we have been using that for long. Even you want to add commands to enable automated builds, it shall be the .config rather than Kconfig that you work on, and the recommended way is to create a brand new .config and use ''make oldconfig'' to generate the full configurations.